ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/carousel/clientlibs/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
```

Comment:
@DeepaliAgrwl  carousel-indicators already present then why "Previous, Next, Pause, play button are visible"?
![image](https://user-images.githubusercontent.com/63295621/81399246-53556680-9148-11ea-97d3-dbef2bb241ac.png)

Reply:
@rakeshkumar-verma  we will be having pause and play functionality as recommended by the reference document